generators/server/templates/gradle/profile_dev.gradle.ejs

```diff
@@ -40,7 +40,8 @@ bootRun {
 }
 
 <%_ if (!skipClient) { _%>
-task webpackBuildDev(type: <%= clientPackageManager.charAt(0).toUpperCase() + clientPackageManager.slice(1) %>Task) {
+task webpackBuildDev(type: <%= _.upperFirst(clientPackageManager) %>Task, dependsOn: '<%= clientPackageManager %>_install') {
+    onlyIf { shouldWebpackRun() == true }
```

Comment:
@atomfrede is this good or is there a better way to do this?

Reply:
Looks ok to me at first sight. Pretty sure one could do something with input and output files and let gradle do the magic to decide wether to run that task or not, but I think thats something for the jhipster plugin (shame on me I didn't had time)

Reply:
I thought of in/out files but figured we would have less control over the behaviour as it would run on change detection as well